mcan/src/reg.rs

```diff
@@ -35,6 +35,12 @@ impl<Id> Can<Id> {
 
 impl<Id: mcan_core::CanId> Can<Id> {
     fn set_init(&self, value: bool) {
+        // Ensure we leave the shutdown state properly if we have entered it.
```

Comment:
`"Power Down" mode`

Reply:
Also, avoid possessive pronouns if possible. E.g.:

`Ensure to leave "Power Down" mode if it was previously entered`

Reply:
Or possibly:

`Ensure "Power Down" mode is exited if it was previously entered`